manager/app/src/main/java/me/weishu/kernelsu/ui/screen/Home.kt

```diff
@@ -100,13 +101,22 @@ fun HomeScreen(navigator: DestinationsNavigator) {
 @Composable
 fun UpdateCard() {
     val context = LocalContext.current
-    val newVersion by produceState(initialValue = Triple(0, "", "")) {
-        value = withContext(Dispatchers.IO) { checkNewVersion() }
+    val latestVersionInfo = LatestVersionInfo(
```

Comment:
we can use default constructor

Reply:
yaa but using a data class should be a better option because data class gives more flexibility i should recommend it

Reply:
data class can have default constructor, you don't need to pass it everytime

Reply:
you can modify this PR directly, i will squash it when merge.